core/primitives/persistent_block/persistent_block.hpp

```diff
@@ -6,10 +6,10 @@
 #ifndef CPP_FILECOIN_CORE_BLOCKCHAIN_BLOCK_PERSISTENT_BLOCK_HPP
 #define CPP_FILECOIN_CORE_BLOCKCHAIN_BLOCK_PERSISTENT_BLOCK_HPP
 
-#include "storage/ipfs/ipfs_block.hpp"
+#include "storage/ipld/ipld_block.hpp"
 
 namespace fc::primitives::blockchain::block {
-  class PersistentBlock : public storage::ipfs::IpfsBlock {
+  class PersistentBlock : public storage::ipld::IPLDBlock {
```

Comment:
Looks like the name is misleading. How persistency is achieved? All the class data is stored in-memory only.

Reply:
Feature _PersistentBlock_ was merged by #89 Feature: chain store by @masterjedy

Reply:
Probably it is not required anymore. I used it before changes in Block were made, making it unnecessary, but I left it just for a case. Maybe I was wrong and should have removed it.

Reply:
PersistentBlock was totally removed